repository/repository.go

```diff
@@ -89,6 +90,7 @@ const (
 	TestPlan90k  PayPlanType = "TEST_PLAN_90K"
 	FreetierV0   PayPlanType = "FREETIER_V0"
 	PayAsYouGoV0 PayPlanType = "PAY_AS_YOU_GO_V0"
+	Enterprise   PayPlanType = "ENTERPRISE"
```

Comment:
This makes sense, i.e. enterprise type plan and corresponding limit.
But makes the `EnterpriseLimit` field more questionable.